src/components/SubTaskList/utils.js

```diff
@@ -1,4 +1,5 @@
-export default (snapValue) => {
+export const getSubtaskListAsArray = (snapValue = {}) => {
+  if (typeof snapValue !== 'object' && snapValue.constructor !== Object) return [];
```

Comment:
What other types can `snapValue` and `snapValue.constructor` be?

FYI, `snapValue` will always  be an object because even if it is an array, arrays are still considered as instances of Object in Javascript and if it is `undefined` it will be an empty object by default.

Reply:
Unless you expect it to be other primitives.

Reply:
Well, a QA engineer walks into a bar... I’ve wrote Unit test that tests what will happen if this function receives anything else except an object and since this function is expected to return an array (at least empty array if there is no subtasks) I need to be sure that there is no way it will return anything except array. Of course, we both know that this function is only used in one place of my code where I pass an object as parameter. But who knows what freak will try to use it in future and what parameters he will try to pass.